extension-manifest-v3/xcode/Ghostery.xcodeproj/project.pbxproj

```diff
@@ -699,6 +701,46 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		46F5023F2B4C55CD0092182B /* Build extension */ = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Build extension";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/zsh;
+			shellScript = "eval \"$(/opt/homebrew/bin/mise activate zsh)\"\ncd ..\nnpm run build -- safari\n";
+		};
+		46F502402B4C59820092182B /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/zsh;
+			shellScript = "eval \"$(/opt/homebrew/bin/mise activate zsh)\"\ncd ..\nnpm run build -- safari\n";
```

Comment:
As re you sure this `/opt/homebrew/bin/mise activate zsh` is a good idea? You assume that environment has homebrew installed, and some `mise` command. Is this required to run npm?